js/index.js

```diff
@@ -0,0 +1,57 @@
+import { navItemsToggle, humburgerToggle } from './navbarToggler.js';
+import data from '../data/projects.js';
+
+// Navbar Toggler
+document.querySelector('.nav-container .checkbox').addEventListener('click', navItemsToggle);
+document.querySelector('.nav-container .checkbox').addEventListener('click', humburgerToggle);
+
+// Modal
+const viewDetailButton = document.querySelectorAll('.btn-view');
+const closeButton = document.querySelectorAll('.btn-close');
+const overlayModal = document.querySelector('.details-modal-overlay');
+const detailsModal = document.querySelector('.details-modal');
+
+closeButton.forEach((button) => {
+  button.addEventListener('click', () => {
+    overlayModal.style.opacity = '0';
+    detailsModal.style.visibility = 'hidden';
+    // Clearing the data from the modal
+    detailsModal.querySelector('.project-title').innerHTML = '';
```

Comment:
- [ ]  Great job on adding the project modals dynamically!. However, to maintain consistency and improve code maintainability, I suggest removing the manually added project cards from the HTML and instead adding them dynamically using JavaScript, just like you did with the modals. This will ensure a more streamlined and efficient implementation.